test.lua

```diff
@@ -274,10 +276,6 @@ local cases = {
         -- macros with mangled names
         ["(require-macros \"test-macros\")\
           (->1 9 (+ 2) (* 11))"]=121,
-        -- macros loaded in function scope shouldn't leak to other functions
-        ["((fn [] (require-macros \"test-macros\") (global x1 (->1 99 (+ 31)))))\
-          (pcall (fn [] (global x1 (->1 23 (+ 1)))))\
-          x1"]=130,
```

Comment:
@technomancy Sorry for not asking this before you merged, but just to be sure:
How come this test was dropped? I'm guessing I'm just not seeing where it's redundant but I wanted to doublecheck just in case it was an accident.

Your updates look great btw!

Reply:
Oh, it wasn't dropped; I just had to move it down to the compiler errors section because I turned strict global checks on. It's just that it won't compile any more.

Reply:
D'oh! I swear I looked for it being added elsewhere. Whoops!